CHANGELOG.md

```diff
@@ -60,7 +66,6 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - Adopted Ruby 2.4+ as the minimum Ruby version in generated extensions
 - Added `bin/console`, `bin/rails` and `bin/setup` to generated extensions
 - Added some Bundler gemspec defaults to generated extensions
-- Configured the default Rake task to run generate the test app before running RSpec
```

Comment:
0.5.0 has already been released, so the changelog shouldn't be changed.

Reply:
Sorry, I started from a revert and this ended up being "below-the-fold" in the diff 😬